public/javascripts/items.js

```diff
@@ -10,25 +10,83 @@ const fetchParams = {
 const params = completed =>
   Object.assign( {}, fetchParams, { body: JSON.stringify({ completed }) })
 
+const params2 = title =>
```

Comment:
`params2` and `params3` jumped out at me as not very intention revealing.  Consider this refactoring to generalize (note that I correct my earlier error of not using CAPS for the constant name):
```js
const params = data => 
  Object.assign( {}, FETCH_PARAMS, { body: JSON.stringify( data ) } )
```
This function will now be called like this:
```js
fetch( `/items/${id}/edit`, params({ title: element[ 0 ].value }) (
```